FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -19,6 +20,50 @@ public final class RemoteFeedLoader: FeedLoader {
 	}
 
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
-		fatalError("Must be implemented")
+		client.get(from: url) { [weak self] result in
+			guard let _ = self else { return }
+			switch result {
+			case let .success((data, response)):
+				guard response.statusCode == RemoteFeedLoader.successStatusCode else {
+					completion(.failure(Error.invalidData))
+					return
+				}
+				guard let feed = try? JSONDecoder().decode(Feed.self, from: data) else {
+					completion(.failure(Error.invalidData))
+					return
+				}
```

Comment:
Consider moving this logic to a helper function, to keep `load` shorter and easier to read.
Also, since both of this `guard` statements complete with the same `Error`, consider joining them:
```suggestion
				guard response.statusCode == RemoteFeedLoader.successStatusCode, 
					  let feed = try? JSONDecoder().decode(Feed.self, from: data) else {
					return completion(.failure(Error.invalidData))
				}
```